app/src/main/java/org/dicio/dicio_android/skills/navigate/NavigateOutput.java

```diff
@@ -0,0 +1,45 @@
+package org.dicio.dicio_android.skills.navigate;
+
+import android.content.Intent;
+import android.net.Uri;
+
+import org.dicio.dicio_android.output.graphical.GraphicalOutputUtils;
+import org.dicio.skill.chain.OutputGenerator;
+
+import java.util.Locale;
+
+public class NavigateOutput extends OutputGenerator<NavigateOutput.Data> {
+
+    public static class Data {
+        public boolean failed = false;
+        public String address;
+    }
+
+    @Override
+    public void generate(final Data data) {
+        if (data.failed) {
+            final String message = "Specify where you want to navigate to";
```

Comment:
You should not hardcode strings to show to the user here, otherwise they can't be localized. Please move them to `strings.xml`